### PR TITLE
Restore parquet meta

### PR DESCRIPTION
**Pull request by fiskus**
I just use parquet loader as a function, then use parquet data in Perspective renderer to optionally display parquet.
Didn't find a way to make code more functional, and gracefully re-use Preview/Display

### CODE REVIEW

**Review by nl0 — COMMENTED** [review]
i'd rather ask @sir-sigurd to add metadata to the tabular preview lambda's response

**Review by nl0 — APPROVED** [review]
looking good in general, some nitpicking inline

File: catalog/app/components/Preview/renderers/ParquetMeta.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ import * as M from '@material-ui/core'
```
[the remark this answers is not in the document]
Comment (fiskus) [reply catalog/app/components/Preview/renderers/ParquetMeta.js:5]: I wasn't sure about that, but I think you're right: it fits to the current system better

File: catalog/app/components/Preview/types.js [hunk 2]
```diff
@@ -4,19 +4,13 @@ import tagged from 'utils/tagged'
 ParquetMeta: {
```
Comment (nl0) [inline catalog/app/components/Preview/types.js:4]: i think you can safely remove `ParquetMeta` *definition* from here, bc it's explicitly present in the `Tabular` loader module

**Pull request merged**